src/info.cr

```diff
@@ -21,7 +21,11 @@ class Shards::Info
     Dir.mkdir_p(@install_path)
 
     unless File.exists?(info_path)
-      Dir[File.join @install_path, "*.sha1"].each { |p| File.delete(p) }
+      Dir.each_child(@install_path) do |name|
```

Comment:
Why is this needed? I thought glob were already supported in windows.

Reply:
No it doesn't work.

https://github.com/crystal-lang/crystal/blob/5999ae29beacf4cfd54e232ca83c1a46b79f26a5/src/dir/glob.cr#L6

Reply:
Also it's a change for the better because now file paths with `*` aren't a hazard

Reply:
Ok, but the culprit for this change is `File.join` then, not `Dir#[]`. I'm ok with the change, but I wanted to know the reason for it.

Reply:
`each_child` is actually better :+1:

In general: glob patterns should just be written as string interpolation, not use `File.join` (you could use `Path.posix` but what's the point?).